### PR TITLE
Improve configuration

### PR DESCRIPTION
**Pull request by derekkraan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,13 +20,9 @@ If you are unfamiliar with these concepts you can catch up with:
...
 
-    Sequent::Core::EventStore.configure do |config|
-      config.event_handlers = [MyEventHandler.new]
-    end
-
-    Sequent::Core::CommandService.configure do |config|
-      config.event_store = Sequent::Core::EventStore.instance
-      config.command_handler_classes = [MyCommandHandler]
+    Sequent.configure do |config|
+      config.event_handlers = [MyEventHandler]
```
Comment (lvonk) [inline README.md]: Shouldn't this be `[MyEventHandler.new]` ?

**Comment by lvonk** [comment]
We should probably bump version to `0.2.0` since it is not backward compatible?

File: README.md (hunk 1 above)
Comment (derekkraan) [inline README.md]: Nope. The `.new` happens elsewhere now.

**Comment by derekkraan** [comment]
IMO that's a decision you would make when deciding to release. Sounds logical though.

File: lib/sequent/core/event_store.rb [hunk 2]
```diff
@@ -72,15 +48,20 @@ def replay_events
...
       protected
+
       def record_class
-        @record_class
+        configuration.record_class
+      end
+
+      def event_handlers
+        configuration.event_handlers
       end
```
Comment (lvonk) [inline lib/sequent/core/event_store.rb]: @derekkraan shouldn't this than be `event_handlers.map(&:new)`?

File: lib/sequent/core/event_store.rb [hunk 3]
```diff
@@ -72,15 +48,20 @@ def replay_events
...
 
       protected
+
       def record_class
-        @record_class
+        configuration.record_class
+      end
+
+      def event_handlers
+        configuration.event_handlers
```
Comment (lvonk) [inline lib/sequent/core/event_store.rb]: @derekkraan shouldn't this than be `event_handlers.map(&:new)`?

File: README.md (hunk 1 above)
Comment (lvonk) [inline README.md]: I can't find that anywhere in the code... See not below. Also in our current implementation we have some `EventHandler`s with collaborators. How would that work in this situation.
I do agree that both `CommandHandler`s and `EventHandler`s should be registered the same way. So either both as instances (my preferences) or both both only the classes. What do you think?

**Comment by derekkraan** [comment]
@lvonk I'm convinced. Any other feedback for this PR?

**Comment by lvonk** [comment]
Nope. Let's merge!

**Pull request merged**